prow/manifests/overlays/metal3/config.yaml

```diff
@@ -415,6 +415,7 @@ presubmits:
             imagePullPolicy: Always
     - name: gomod
       branches:
+        - release-0.6
```

Comment:
Put main on top please on each of these. We'll add the releases beneath as needed.

Reply:
done